src/main/kotlin/baseball/Application.kt

```diff
@@ -1,5 +1,111 @@
 package baseball
 
+import camp.nextstep.edu.missionutils.Console
+import camp.nextstep.edu.missionutils.Randoms
+
+enum class PlayerInputType(val type: Int) {
+    GAME(3), // 3 자리 수 입력 (플레이어의 숫자)
+    REPLAY(1) // 1 자리 수 입력 (게임 재시작 여부)
+}
+
 fun main() {
-    TODO("프로그램 구현")
+
+    var play = true // 게임 진행 결정
+
+    val computerList = mutableListOf<Int>() // 컴퓨터의 숫자를 담을 리스트
+    val playerList = mutableListOf<Int>() // 플레이어의 숫자를 담을 리스트
+    var playerString: String? // 플레이어의 입력을 받아 playerList에 담아줌
+
+    println("숫자 야구 게임을 시작합니다.")
+
+    while (play) {
+        generateComputerList(computerList) // 컴퓨터의 숫자 생성
+
+        while (true) {
+            print("숫자를 입력해주세요 : ")
+            playerString = Console.readLine()
```

Comment:
아래와 같이 하면 playerString에 null이 들어갈 경우가 없을 것 같습니다.
```suggestion
            playerString = Console.readLine() ?: throw IllegalArgumentException()
```

Reply:
아 맞네요.. 코드 예시까지 감사합니다!!